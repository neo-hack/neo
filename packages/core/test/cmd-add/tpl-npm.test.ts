import {
  execNeo,
  randomStoreDir,
  readLockFile,
} from '../helpers'

const storeDir: string = randomStoreDir()

it('load template from npm should work', async () => {
  await execNeo(['add', '@aiou/ts-lib-template', '--store-dir', storeDir], {
    env: {
      DEBUG: 'neo:*',
    },
    stderr: 'inherit',
    stdout: 'inherit',
  })
  expect(readLockFile(storeDir)).toMatchSnapshot()
}, 10000)
