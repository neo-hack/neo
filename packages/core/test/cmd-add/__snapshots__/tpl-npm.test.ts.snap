// Vitest Snapshot v1

exports[`load template from npm should work 1`] = `
{
  "templates": {
    "@aiou/ts-lib-template/0.6.0": {
      "id": "@aiou/ts-lib-template/0.6.0",
      "name": "@aiou/ts-lib-template",
      "pref": "@aiou/ts-lib-template",
      "version": "0.6.0",
    },
  },
  "version": undefined,
}
`;
